### DISCUSSION TITLE
What is the ways of Backup and Restore the Netbox tool and the Data

### DISCUSSION THREAD

**Discussion by mahmoudkhalil2** [netbox-community/netbox#13562] [Q&A]
Hello,
please let me know what is the available ways of Backup and Restore the Netbox tool itself and the Data imported to it, in case any issues or corruption happened to the Netbox tool itself or the tool is fine but the data was deleted or corrupted to have ways to restore the tool or the data or both.

Best Regards

**Comment by deku-m** [comment]
I think this is your answer:
https://docs.netbox.dev/en/stable/administration/replicating-netbox/

A weekly/daily script/cronjob will do the trick storing to external source.
